migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/sql_schema_differ_flavour/postgres.rs

```diff
@@ -22,6 +22,8 @@ static POSTGIS_TABLES_OR_VIEWS: Lazy<RegexSet> = Lazy::new(|| {
         "(?i)^spatial_ref_sys$",
         "(?i)^geometry_columns$",
         "(?i)^geography_columns$",
+        "(?i)^raster_columns$",
+        "(?i)^raster_overviews$",
```

Comment:
These tables aren't in the link at the top of the array. Is there a postgis doc page that mentions them? (it does look legit, just that we should document where we got the names)

Reply:
Added a link that mentions those 2 columns

Reply:
Awesome, thanks